__tests__/app.test.js

```diff
@@ -242,6 +262,30 @@ test("400: Sends an appropriate status and error message when given an invalid o
     .then(({ body }) => {
       expect(body.msg).toBe('Invalid order query');
     });
+});
+test("400: Sends an appropriate status and error message when given an invalid topic query", () => {
+  return request(app)
+    .get('/api/articles?topic=hi')
+    .expect(400)
+    .then(({ body }) => {
+      expect(body.msg).toBe('Invalid topic query');
+    });
+});
+test("400: Should respond with appropriate error messages in order when given multiple invalid queries", () => {
```

Comment:
You don't _have_ to be too explicit with error messages, if it's a bad request then that's all you need to say. Don't forget that you will have the endpoints document/endpoint that will list all the valid queries for the user